README.md

```diff
@@ -0,0 +1,26 @@
+Usage:
```

Comment:
- [ ] Add package title
- [ ] Add short description of the module
- [ ] Add an installation section
- [ ] Notes about compatibility. Does this package *require* the Postgres database? Or is it compatible with any DB?
- [ ] Notes on how to get your Redis config or a link to some compatible setup guide?

````suggestion
# @boardgame.io/redis-pubsub

This package provides a Redis pub/sub adapter for [boardgame.io](https://boardgame.io/).

## Installation

```sh
npm install @boardgame.io/redis-pubsub
```

## Usage
````

Reply:
Done.